app/assets/javascripts/components/PipelineSampleReport.jsx

```diff
@@ -593,6 +593,7 @@ class PipelineSampleReport extends React.Component {
   //Download report in csv
   downloadReport(id) {
     location.href = `/reports/${id}/csv`
+    _satellite.track('downloadreport')
```

Comment:
this line should be moved above the location.href, otherwise it might not be triggered.